scripts/tests/ec2-memory-benchmark/ec2-memory-benchmark.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh 
+dataverse_branch=$1
+
+if [ $dataverse_branch"x" = "x" ]
+then
+    echo "usage: ./ec2-memory-benchmark.sh <dataverse_branch>"
+    exit 1
+fi
+
+# download the ec2-create-instance script:
+curl -O https://raw.githubusercontent.com/IQSS/dataverse-ansible/master/ec2/ec2-create-instance.sh
+chmod 755 ec2-create-instance.sh
+
+# download the sample data ec2 config: 
+curl -O https://raw.githubusercontent.com/IQSS/dataverse-sample-data/memory-monitoring-experiments/ec2config_memory_test.yaml
+
+# run the script: 
+./ec2-create-instance.sh -a ec2-memory-benchmark -b ${dataverse_branch} -r https://github.com/IQSS/dataverse.git -g ec2config_memory_test.yaml 2>&1 | tee create-instance.log
```

Comment:
Instead of hard coding the IQSS repo here, should we allow any fork to be specified? We could always add this later, of course.

Reply:
Right now it's not going to work with any other branch. On account of the bombing external tools, for example. We'll want to add this later, yes.

Reply:
Ok, sounds good.